src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -254,6 +267,10 @@ async def genomic_to_transcript_exon_coordinates(
             following transcripts: MANE Select, MANE Clinical Plus, Longest Remaining
             Compatible Transcript. See the :ref:`Transcript Selection policy <transcript_selection_policy>`
             page.
+        :param get_nearest_transcript_junction: If the exon that is adjacent to a breakpoint should be returned. For the positive strand, adjacent is defined
+        as the exon preceeding the breakpoint for the 5' end and the exon following the breakpoint for the 3' end.
+        For the negative strand, adjacent is defined as the exon following the breakpoint for the 5' end and the exon
+        preceeding the breakpoint for the 3' end. If ``True``, this will report the adjacent exon if the position specified by``start`` or ``end`` does not occur on an exon.
```

Comment:
I still don't know if this makes sense `If the exon that is adjacent to a breakpoint should be returned`. `preceding` was also misspelled. The formatting is also inconsistent. I would suggest changing to:

```suggestion
        :param get_nearest_transcript_junction: If ``True``, this will return the
            adjacent exon if the position specified by``start`` or ``end`` does not
            occur on an exon. For the positive strand, adjacent is defined as the exon
            preceding the breakpoint for the 5' end and the exon following the
            breakpoint for the 3' end. For the negative strand, adjacent is defined as
            the exon following the breakpoint for the 5' end and the exon preceding the
            breakpoint for the 3' end.
```